cpp/installer.cc

```diff
@@ -492,10 +492,22 @@ void InstallUBTools(const bool make_install, const OSSystemType os_system_type)
         ExecUtil::ExecOrDie(ExecUtil::LocateOrDie("git"), { "clone", git_url, ZOTERO_ENHANCEMENT_MAPS_DIRECTORY });
     }
 
+    // logfile for zts docker container
+    const std::string ZTS_LOGFILE(UBTools::GetTueFindLogPath() + "/zts.log");
+    if (not FileUtil::Exists(ZTS_LOGFILE))
+        FileUtil::WriteStringOrDie(ZTS_LOGFILE, "");
+    if (os_system_type == UBUNTU) {
+        // This is only necessary for UBUNTU since syslogd does not run with root privileges.
+        ExecUtil::ExecOrDie(ExecUtil::LocateOrDie("chown"), { "syslog:adm", ZTS_LOGFILE });
```

Comment:
Could be rolled into a possible "Touch" function.

Reply:
Since "Touch" implicates that the function has a similar purpose like the bash command, i'd rather create a FileUtil::ChangeOwnerOrDie function.